documentation/src/main/wiki/AETIn10Minutes.md

```diff
@@ -54,55 +38,25 @@ Create the file named `suite.xml` with the following content:
 </suite>
 ```
 
-Then create another file named `pom.xml` with the following content:
-
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
-    <groupId>test-group</groupId>
-    <artifactId>test-project</artifactId>
-    <version>1.0.0</version>
-    <packaging>pom</packaging>
-
-    <name>Test project</name>
-    <url>http://www.example.com</url>
-
-    <properties>
-        <aet.version>2.1.5</aet.version>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    </properties>
-
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>com.cognifide.aet</groupId>
-                <artifactId>aet-maven-plugin</artifactId>
-                <version>${aet.version}</version>
-            </plugin>
-        </plugins>
-    </build>
-</project>
-```
-
-It does not need to be in the same directory as the `suite.xml` file.
+Then download the AET [shell script](https://github.com/Cognifide/aet/blob/master/client/client-scripts/aet.sh)
+(see required prerequisites in [README](https://github.com/Cognifide/aet/blob/master/client/client-scripts/README.md)).
 
 ### Run the Test
 
-Once you have created both `suite.xml` and `pom.xml` files open a command prompt in the directory which contains the `pom.xml` file and execute the following command:
-
+Once you have created both `suite.xml` and downloaded `aet.sh`, open a command prompt in the directory which contains the `aet.sh` file and execute the following command:
 ```
-mvn aet:run -DtestSuite=full/path/to/suite.xml
+./aet.sh http://localhost:8181 path/to/suite.xml
 ```
-
 Remember to provide the path to your `suite.xml` file.
```

Comment:
If you put `aet.sh` and `suite.xml` in the same directory, you don't have to explicitly pass the `path/to/suite.xml` parameter. By default `aet.sh` (as well as aet-maven-plugin) searches for the file named `suite.xml` in the directory.

Reply:
Updated